includes/polling/netstats-snmp.inc.php

```diff
@@ -54,7 +54,7 @@
             $value = $data_array[0][$oid];
         }
         else {
-            $value = 'U';
+            $value = null;
```

Comment:
In that case either revert the change and leave it as it was or remove the code if it's not needed.